ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -153,9 +153,12 @@ def get_dbt_owner(
             if manifest_node:
                 dbt_owner = manifest_node.meta.get(DbtCommonEnum.OWNER.value)
             if dbt_owner:
-                owner = self.metadata.get_reference_by_name(
-                    name=dbt_owner, is_owner=True
-                )
+                if self.source_config.dbtOwnerUseEmail:
```

Comment:
hi @aballiet, thanks for the PR. As discussed in Slack, we could use a generic fallback of fetching mail by ES instead of having an explicit config to keep it simpler. Thanks!!